ArmTemplates/app-service.json

```diff
@@ -52,6 +52,10 @@
         "description": "Format removing backlashes: [{\"Name\": \"IP A\",\"ipAddress\": \"123.123.123.123\"},{\"Name\": \"IP B\",\"ipAddress\": \"234.234.234.234\"}]"
       }
     },
+    "ipSecurityRestrictionsDefaultAction": {
+      "type": "string",
+      "defaultValue": "Deny"
```

Comment:
Please change the default value to "Allow", if you set it to "Deny" once merged in new deployments to resac and mcom will start blocking traffic